README.md

```diff
@@ -135,216 +135,216 @@ See file output (this is just part of the file):
+/intel/osv/trace/vfs/vfs_fchdir| uint64|
+/intel/osv/trace/vfs/vfs_fchdir_ret| uint64|
+/intel/osv/trace/vfs/vfs_fchdir_err| uint64|
+/intel/osv/trace/vfs/vfs_link| uint64|
+/intel/osv/trace/vfs/vfs_link_ret| uint64|
+/intel/osv/trace/vfs/vfs_link_err| uint64|
+/intel/osv/trace/vfs/vfs_symlink| uint64|
+/intel/osv/trace/vfs/vfs_symlink_ret| uint64|
+/intel/osv/trace/vfs/vfs_symlink_err| uint64|
+/intel/osv/trace/vfs/vfs_unlink| uint64|
+/intel/osv/trace/vfs/vfs_unlink_ret| uint64|
+/intel/osv/trace/vfs/vfs_unlink_err| uint64|
+/intel/osv/trace/vfs/vfs_stat| uint64|
+/intel/osv/trace/vfs/vfs_stat_ret| uint64|
+/intel/osv/trace/vfs/vfs_stat_err| uint64|
+/intel/osv/trace/vfs/vfs_lstat| uint64|
+/intel/osv/trace/vfs/vfs_lstat_ret| uint64|
+/intel/osv/trace/vfs/vfs_lstat_err| uint64|
+/intel/osv/trace/vfs/vfs_statfs| uint64|
+/intel/osv/trace/vfs/vfs_statfs_ret| uint64|
+/intel/osv/trace/vfs/vfs_statfs_err| uint64|
+/intel/osv/trace/vfs/vfs_fstatfs| uint64|
+/intel/osv/trace/vfs/vfs_fstatfs_ret| uint64|
+/intel/osv/trace/vfs/vfs_fstatfs_err| uint64|
+/intel/osv/trace/vfs/vfs_getcwd| uint64|
+/intel/osv/trace/vfs/vfs_getcwd_ret| uint64|
+/intel/osv/trace/vfs/vfs_getcwd_err| uint64|
+/intel/osv/trace/vfs/vfs_dup| uint64|
+/intel/osv/trace/vfs/vfs_dup_ret| uint64|
+/intel/osv/trace/vfs/vfs_dup_err| uint64|
+/intel/osv/trace/vfs/vfs_dup3| uint64|
+/intel/osv/trace/vfs/vfs_dup3_ret| uint64|
+/intel/osv/trace/vfs/vfs_dup3_err| uint64|
+/intel/osv/trace/vfs/vfs_fcntl| uint64|
+/intel/osv/trace/vfs/vfs_fcntl_ret| uint64|
+/intel/osv/trace/vfs/vfs_fcntl_err| uint64|
+/intel/osv/trace/vfs/vfs_access| uint64|
+/intel/osv/trace/vfs/vfs_access_ret| uint64|
+/intel/osv/trace/vfs/vfs_access_err| uint64|
+/intel/osv/trace/vfs/vfs_isatty| uint64|
+/intel/osv/trace/vfs/vfs_isatty_ret| uint64|
+/intel/osv/trace/vfs/vfs_isatty_err| uint64|
+/intel/osv/trace/vfs/vfs_truncate| uint64|
+/intel/osv/trace/vfs/vfs_truncate_ret| uint64|
+/intel/osv/trace/vfs/vfs_truncate_err| uint64|
+/intel/osv/trace/vfs/vfs_ftruncate| uint64|
+/intel/osv/trace/vfs/vfs_ftruncate_ret| uint64|
+/intel/osv/trace/vfs/vfs_ftruncate_err| uint64|
+/intel/osv/trace/vfs/vfs_fallocate| uint64|
+/intel/osv/trace/vfs/vfs_fallocate_ret| uint64|
+/intel/osv/trace/vfs/vfs_fallocate_err| uint64|
+/intel/osv/trace/vfs/vfs_utimes| uint64|
+/intel/osv/trace/vfs/vfs_utimes_ret| uint64|
+/intel/osv/trace/vfs/vfs_utimes_err| uint64|
+/intel/osv/trace/vfs/vfs_utimensat| uint64|
+/intel/osv/trace/vfs/vfs_utimensat_ret| uint64|
+/intel/osv/trace/vfs/vfs_utimensat_err| uint64|
+/intel/osv/trace/vfs/vfs_futimens| uint64|
+/intel/osv/trace/vfs/vfs_futimens_ret| uint64|
+/intel/osv/trace/vfs/vfs_futimens_err| uint64|
+/intel/osv/trace/vfs/vfs_chmod| uint64|
+/intel/osv/trace/vfs/vfs_chmod_ret| uint64|
+/intel/osv/trace/vfs/vfs_chmod_err| uint64|
+/intel/osv/trace/vfs/vfs_fchmod| uint64|
+/intel/osv/trace/vfs/vfs_fchmod_ret| uint64|
+/intel/osv/trace/vfs/vfs_fchown| uint64|
+/intel/osv/trace/vfs/vfs_fchown_ret| uint64|
 
```

Comment:
The list of metrics is long, making it difficult to read all of the README. Consider moving to external document, METRICS.MD, see [meminfo collector](https://github.com/intelsdi-x/snap-plugin-collector-meminfo#collected-metrics) for example.